pkgs/by-name/on/oneanime/package.nix

```diff
@@ -134,60 +125,58 @@ flutter329.buildFlutterApplication rec {
       };
   };
 
-  desktopItems = [
-    (makeDesktopItem {
-      name = "oneanime";
-      exec = "oneanime";
-      icon = "oneanime";
-      desktopName = "oneAnime";
-    })
-  ];
-
   nativeBuildInputs = [
     autoPatchelfHook
-    copyDesktopItems
+    desktop-file-utils
   ];
 
   buildInputs = [
     alsa-lib
-    libepoxy
-    libpulseaudio
     libdrm
+    libepoxy
     libgbm
-    mpv
+    libpulseaudio
+    mpv-unwrapped
   ];
 
-  postPatch = ''
-    substituteInPlace lib/pages/init_page.dart \
-      --replace-fail "lib/opencc.so" "${libopencc}/lib/libopencc.so"
-  '';
-
   postInstall = ''
-    ln -snf ${mpv}/lib/libmpv.so.2 $out/app/oneanime/lib/libmpv.so.2
-    install -Dm0644 assets/images/logo/logo_android_2.png  $out/share/pixmaps/oneanime.png
+    ln --symbolic --no-dereference --force ${mpv-unwrapped}/lib/libmpv.so.2 $out/app/oneanime/lib/libmpv.so.2
```

Comment:
There is no point in expanding very well known arguments of well known commands to longer variants that probably was less people know.